dask_expr/io/tests/test_io.py

```diff
@@ -495,3 +496,12 @@ def test_from_pandas_divisions_duplicated():
     assert_eq(df.partitions[0], pdf.loc[1:4])
     assert_eq(df.partitions[1], pdf.loc[5:6])
     assert_eq(df.partitions[2], pdf.loc[8:])
+
+
+def test_from_dask_array():
+    import dask.array as da
+
+    arr = da.ones((20, 4), chunks=(2, 2))
+    df = from_dask_array(arr, columns=["a", "b", "c", "d"])
+    pdf = lib.DataFrame(arr.compute(), columns=list("abcd"))
```

Comment:
nit: Let's be as consistent as possible
```suggestion
    pdf = lib.DataFrame(arr.compute(), columns=["a", "b", "c", "d"])
```

Reply:
got tired of typing :)